### PR TITLE
Fixes #1631: Gender icon added for User Profile

### LINKED ISSUE #1631: Inappropriate icon for gender in User Profile
**Summary:**
Icon meant for groups is used for gender in User Profile.

**Expected behaviour:**
New icon for gender 

**Device and Android version:**
Any

**Screenshots:**
ss

### PR DESCRIPTION
**Pull request by PatelVatsalB21**
Fixes #1631 

New icon for gender in User Profile added

ss

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by PatelVatsalB21** [comment]
@garvit984, @miPlodder please review this pr

File: app/src/main/res/drawable/ic_gender_24dp.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<vector android:height="24dp" android:viewportHeight="512"
```
Comment (miPlodder) [inline app/src/main/res/drawable/ic_gender_24dp.xml:1]: Are we using standard drawable? Can you clarify where it's taken? 
The rest looks good to me.

**Review by miPlodder — APPROVED** [review]

File: app/src/main/res/drawable/ic_gender_24dp.xml (hunk 1 above)
Comment (PatelVatsalB21) [reply app/src/main/res/drawable/ic_gender_24dp.xml:1]: @miPlodder I didn't understood what are you talking about, but i just downloaded svg icon and imported vector asset in drawable. Should I change anything or is this ok?

Comment (miPlodder) [reply app/src/main/res/drawable/ic_gender_24dp.xml:1]: I'm just asking where this SVG has been taken from? (just share the link)

Comment (PatelVatsalB21) [reply app/src/main/res/drawable/ic_gender_24dp.xml:1]: [gender icon](https://www.flaticon.com/free-icon/toilet-signs_3917918?term=gender&page=1&position=58&related_item_id=3917918)
i edited it to match app theme

**Review by miPlodder — APPROVED** [review]
Good job

**Pull request merged**